ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+            public level: number;
+            public options: BreadcrumbsOptions;
+
+            public alignGroup(): void;
+            public createList(e?: any): void;
+            public destroy(this: Breadcrumbs, lastVisibleLevel: number|undefined): void;
+            public draw(): void;
+            public multipleDrillUp(numb: number): void;
+            public init(chart: Chart, userOptions: Highcharts.Options): void;
+            public redraw(): void;
+            public renderButton(breadcrumb: Array<any>, posX: number, posY: number): SVGElement|void;
+            public renderSeparator(posX: number, posY: number): SVGElement|void;
+            public update(userOptions: Highcharts.Options, redrawFlag: boolean): void;
+        }
+    }
+}
+
+// Add language support.
+extend(
+    defaultOptions.lang,
+    /**
+     * @optionparent lang
+     */
+    {
+        /**
+         * @since    next
+         * @product  highcharts
+         */
+        mainBreadCrumb: 'Main'
+    }
+);
+
+extend(
+    defaultOptions,
+    {
+        /**
+         * Options for breadcrumbs
+         *
+         * @since   next
+         * @product highcharts
+         */
+        breadcrumbs: {
+            /**
+            * The default padding for each button and separator.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            buttonPadding: 5,
+
+            /**
+            * Show only last button instead of whole drillUp tree.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            showOnlyLast: true,
+
+            /**
+            * Enable or disable the breadcrumbs.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            enabled: true,
+
+            /**
+            * Fires when clicking on the breadcrumbs button.
+            * Two arguments are passed to the function. First breadcrumb button
+            * as an SVG element. Second is the breadcrumbs class,
+            * containing reference to the chart, series etc.
+            *
+            * ```js
+            * click: function(button, breadcrumbs) {
+            *   console.log(button);
+            * }
+            * ```
+            *
+            * Return false to stop default button's click action. // TO DO
+            *
+            * @type      {Highcharts.DrilldownBreadcrumbsClickCallbackFunction}
+            * @since     next
+            */
+
+            /**
+            * When the breadcrumbs is floating, the plot area will not move
+            * to make space for it. By default, the chart will not make space
+            * for the buttons.
+            * This property won't work when positioned in the middle.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            floating: true,
+
+            /**
+            * A format string for the breadcrumbs button.
```

Comment:
Could you improve a little this description? For example, what is the context for this format? Is that a `point.value`?